### DISCUSSION TITLE
Disabled repository on bitbucket

### DISCUSSION THREAD

**Discussion by amitnirG** [renovatebot/renovate#16064] [Request Help] [answered]
Hey,

I have configured renovate on my private Bitbucket repo and everything seemed to work fine but I accidentally set the wrong branch as the main branch - so I've closed the PR that was opened by the renovate-bot so I could re-configure it.
The problem is that It's not possible to rename a closed PR on Bitbucket Cloud  - so the repo is now disabled without a possibility to restart the onboarding process.
![image](https://user-images.githubusercontent.com/86183794/173533009-f2dd99ac-b6bd-4647-b1a9-eb5ef18ebd5e.png)

I have tried to commit a new config to the main branch but it didn't seem to work.

Is there a possibility you could open it manually, please?

**Comment by rarkins** [comment, marked as answer]
If you manually add a `renovate.json` to the repo's default branch then it should work. Do that and then run with `LOG_LEVEL=debug` to see what it says.

**Comment by amitnirG** [comment]
Tried that before - but now tried again and it seemed to work :)
Thank you
